go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/Nordix/simple-ipam v1.0.0
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/emicklei/go-restful v2.15.0+incompatible
+	github.com/envoyproxy/data-plane-api v0.0.0-20211102050950-ee95d36f0cd6 // indirect
```

Comment:
Did this come up from running the `make` targets?

Reply:
Ho didn't notice this let me check